source/_components/roku.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: "Roku"
+description: "Instructions how to integrate Roku devices into Home Assistant."
+date: 2017-02-19 20:47
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: roku.png
+ha_category: Hub
+ha_iot_class: "Local Push"
+ha_release: 0.64
+---
+
+The [Roku](http://www.roku.com/) platform allows integration of Roku [media players](/components/media_player.roku) and [remotes](/components/remote.roku/) which will be automatically discovered if you enable the [discovery component](/components/discovery/).
+
+<p class='note'>
+Due to changes in the autodiscovery mechanism, Roku media player entity names will not match prior releases before 0.64. 
+</p>
+
+The `roku` platform can also be forced to load by adding the following lines to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+roku:
+  host: 192.168.1.100
+  name: Roku
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

Reply:
Done.